src/panels/Scene/SceneTree/SceneTree.tsx

```diff
@@ -27,7 +27,11 @@ import {
   sortTreeData
 } from './treeUtil';
 
-export function SceneTree() {
+interface Props {
+  siblingRef: React.RefObject<HTMLDivElement>;
```

Comment:
Is this the best name for this variable?  Maybe give it a comment that explains why it is the way it is?

Reply:
ALternatively, move the computeHeight function to the Scene.tsx file and pass the function instead of the ref?  Avoids having to look in another file to see what this ref actually is